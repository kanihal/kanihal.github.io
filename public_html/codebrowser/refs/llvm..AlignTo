<def f='include/llvm-7/llvm/Support/MathExtras.h' l='708' ll='714'/>
