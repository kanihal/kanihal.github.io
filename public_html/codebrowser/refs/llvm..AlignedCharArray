<def f='include/llvm-7/llvm/Support/AlignOf.h' l='36' ll='38'/>
<use f='include/llvm-7/llvm/Support/AlignOf.h' l='138'/>
<mbr r='llvm::AlignedCharArray::buffer' t='char [Size]'/>
<use f='include/llvm-7/llvm/Support/Endian.h' l='246'/>
<use f='include/llvm-7/llvm/Support/TrailingObjects.h' l='373'/>
