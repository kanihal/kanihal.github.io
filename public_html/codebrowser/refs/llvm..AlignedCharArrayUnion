<def f='include/llvm-7/llvm/Support/AlignOf.h' l='138' ll='143'/>
<use f='include/llvm-7/llvm/ADT/Optional.h' l='33'/>
<use f='include/llvm-7/llvm/ADT/Optional.h' l='114'/>
<use f='include/llvm-7/llvm/ADT/SmallVector.h' l='75'/>
<use f='include/llvm-7/llvm/ADT/SmallVector.h' l='76'/>
<use f='include/llvm-7/llvm/ADT/SmallVector.h' l='830'/>
<use f='include/llvm-7/llvm/ADT/DenseMap.h' l='826'/>
<use f='include/llvm-7/llvm/ADT/DenseMap.h' l='969' c='_ZN4llvm13SmallDenseMap4growEj'/>
<use f='include/llvm-7/llvm/Support/ErrorOr.h' l='263'/>
<use f='include/llvm-7/llvm/Support/ErrorOr.h' l='264'/>
<use f='include/llvm-7/llvm/Support/Error.h' l='670'/>
<use f='include/llvm-7/llvm/Support/Error.h' l='671'/>
