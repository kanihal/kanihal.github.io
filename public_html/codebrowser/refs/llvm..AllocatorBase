<def f='include/llvm-7/llvm/Support/Allocator.h' l='46' ll='91'/>
<ovr f='include/llvm-7/llvm/Support/Allocator.h' l='93' c='llvm::MallocAllocator'/>
<use f='include/llvm-7/llvm/Support/Allocator.h' l='93'/>
<use f='include/llvm-7/llvm/Support/Allocator.h' l='103'/>
<use f='include/llvm-7/llvm/Support/Allocator.h' l='110'/>
<use f='include/llvm-7/llvm/Support/Allocator.h' l='141'/>
<use f='include/llvm-7/llvm/Support/Allocator.h' l='272'/>
<use f='include/llvm-7/llvm/Support/Allocator.h' l='282'/>
<doc f='include/llvm-7/llvm/Support/Allocator.h' l='40'>/// CRTP base class providing obvious overloads for the core \c
/// Allocate() methods of LLVM-style allocators.
///
/// This base class both documents the full public interface exposed by all
/// LLVM-style allocators, and redirects all of the overloads to a single core
/// set of methods which the derived class must define.</doc>
<fun r='_ZN4llvm13AllocatorBase8AllocateEmm'/>
<fun r='_ZN4llvm13AllocatorBase10DeallocateEPKvm'/>
<fun r='_ZN4llvm13AllocatorBase8AllocateEm'/>
<fun r='_ZN4llvm13AllocatorBase10DeallocateEPT_m'/>
