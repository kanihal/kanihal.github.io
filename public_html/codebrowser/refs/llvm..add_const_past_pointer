<def f='include/llvm-7/llvm/Support/type_traits.h' l='89'/>
<def f='include/llvm-7/llvm/Support/type_traits.h' l='91' ll='95'/>
<use f='include/llvm-7/llvm/Support/type_traits.h' l='104'/>
<use f='include/llvm-7/llvm/Support/Casting.h' l='44'/>
<use f='include/llvm-7/llvm/ADT/SmallPtrSet.h' l='345'/>
