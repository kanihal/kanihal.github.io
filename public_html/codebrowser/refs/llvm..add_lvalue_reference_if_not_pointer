<def f='include/llvm-7/llvm/Support/type_traits.h' l='78'/>
<def f='include/llvm-7/llvm/Support/type_traits.h' l='80' ll='84'/>
<use f='include/llvm-7/llvm/Support/Casting.h' l='46'/>
