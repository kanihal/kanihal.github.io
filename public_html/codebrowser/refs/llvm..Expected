<def f='include/llvm-7/llvm/Support/Error.h' l='427' ll='677'/>
<use f='include/llvm-7/llvm/Support/Error.h' l='486' c='_ZN4llvm8ExpectedC1EONS_8ExpectedIT_EEPNSt9enable_ifIXsr3std14is_convertibleIS2_T_EE5valueEvE4typeE'/>
<use f='include/llvm-7/llvm/Support/Error.h' l='496' c='_ZN4llvm8ExpectedC1EONS_8ExpectedIT_EEPNSt9enable_ifIXntsr3std14is_convertibleIS2_T_EE5valueEvE4typeE'/>
<use f='include/llvm-7/llvm/Support/Error.h' l='534' c='_ZNK4llvm8Expected3getEv'/>
<use f='include/llvm-7/llvm/Support/Error.h' l='588' c='_ZN4llvm8Expected13moveConstructEONS_8ExpectedIT_EE'/>
<use f='include/llvm-7/llvm/Support/Error.h' l='601' c='_ZN4llvm8Expected10moveAssignEONS_8ExpectedIT_EE'/>
<use f='include/llvm-7/llvm/Support/Error.h' l='719' c='_ZN4llvm8cantFailENS_8ExpectedIT_EEPKc'/>
<use f='include/llvm-7/llvm/Support/Error.h' l='743' c='_ZN4llvm8cantFailENS_8ExpectedIRT_EEPKc'/>
<use f='include/llvm-7/llvm/Support/Error.h' l='933' c='_ZN4llvm14handleExpectedENS_8ExpectedIT_EEOT0_DpOT1_'/>
<use f='include/llvm-7/llvm/Support/Error.h' l='933' c='_ZN4llvm14handleExpectedENS_8ExpectedIT_EEOT0_DpOT1_'/>
<use f='include/llvm-7/llvm/Support/Error.h' l='1035' c='_ZN4llvm22ExpectedAsOutParameterC1EPNS_8ExpectedIT_EE'/>
<use f='include/llvm-7/llvm/Support/Error.h' l='1047'/>
<use f='include/llvm-7/llvm/Support/Error.h' l='1091' c='_ZN4llvm17errorOrToExpectedEONS_7ErrorOrIT_EE'/>
<use f='include/llvm-7/llvm/Support/Error.h' l='1098' c='_ZN4llvm17expectedToErrorOrEONS_8ExpectedIT_EE'/>
<use f='include/llvm-7/llvm/Support/Error.h' l='1161' c='_ZNK4llvm11ExitOnErrorclEONS_8ExpectedIT_EE'/>
<use f='include/llvm-7/llvm/Support/Error.h' l='1168' c='_ZNK4llvm11ExitOnErrorclEONS_8ExpectedIRT_EE'/>
<use f='include/llvm-7/llvm/Support/FileSystem.h' l='781' c='_ZN4llvm3sys2fs8TempFile6createERKNS_5TwineEj'/>
<use f='include/llvm-7/llvm/Support/FileSystem.h' l='899' c='_ZN4llvm3sys2fs14openNativeFileERKNS_5TwineENS1_19CreationDispositionENS1_10FileAccessENS1_9OpenFlagsEj'/>
<use f='include/llvm-7/llvm/Support/FileSystem.h' l='938' c='_ZN4llvm3sys2fs22openNativeFileForWriteERKNS_5TwineENS1_19CreationDispositionENS1_9OpenFlagsEj'/>
<use f='include/llvm-7/llvm/Support/FileSystem.h' l='980' c='_ZN4llvm3sys2fs26openNativeFileForReadWriteERKNS_5TwineENS1_19CreationDispositionENS1_9OpenFlagsEj'/>
<use f='include/llvm-7/llvm/Support/FileSystem.h' l='1017' c='_ZN4llvm3sys2fs21openNativeFileForReadERKNS_5TwineENS1_9OpenFlagsEPNS_15SmallVectorImplIcEE'/>
<doc f='include/llvm-7/llvm/Support/Error.h' l='421'>/// Tagged union holding either a T or a Error.
///
/// This class parallels ErrorOr, but replaces error_code with Error. Since
/// Error cannot be copied, this class replaces getError() with
/// takeError(). It also adds an bool errorIsA&lt;ErrT&gt;() method for testing the
/// error class type.</doc>
<smbr r='llvm::Expected::isRef' t='const bool'/>
<fun r='_ZN4llvm8ExpectedC1ENS_5ErrorE'/>
<fun r='_ZN4llvm8ExpectedC1ENS_12ErrorSuccessE'/>
<fun r='_ZN4llvm8ExpectedC1EOT_PNSt9enable_ifIXsr3std14is_convertibleIS1_T_EE5valueEvE4typeE'/>
<fun r='_ZN4llvm8ExpectedC1EONS_8ExpectedIT_EE'/>
<fun r='_ZN4llvm8ExpectedC1EONS_8ExpectedIT_EEPNSt9enable_ifIXsr3std14is_convertibleIS2_T_EE5valueEvE4typeE'/>
<fun r='_ZN4llvm8ExpectedC1EONS_8ExpectedIT_EEPNSt9enable_ifIXntsr3std14is_convertibleIS2_T_EE5valueEvE4typeE'/>
<fun r='_ZN4llvm8ExpectedaSEONS_8ExpectedIT_EE'/>
<fun r='_ZN4llvm8ExpectedD1Ev'/>
<fun r='_ZN4llvm8ExpectedcvbEv'/>
<fun r='_ZN4llvm8Expected3getEv'/>
<fun r='_ZNK4llvm8Expected3getEv'/>
<fun r='_ZNK4llvm8Expected8errorIsAEv'/>
<fun r='_ZN4llvm8Expected9takeErrorEv'/>
<fun r='_ZN4llvm8ExpectedptEv'/>
<fun r='_ZNK4llvm8ExpectedptEv'/>
<fun r='_ZN4llvm8ExpecteddeEv'/>
<fun r='_ZNK4llvm8ExpecteddeEv'/>
<fun r='_ZN4llvm8Expected21compareThisIfSameTypeERKT_S3_'/>
<fun r='_ZN4llvm8Expected21compareThisIfSameTypeERKT_RKT0_'/>
<fun r='_ZN4llvm8Expected13moveConstructEONS_8ExpectedIT_EE'/>
<fun r='_ZN4llvm8Expected10moveAssignEONS_8ExpectedIT_EE'/>
<fun r='_ZN4llvm8Expected9toPointerEPNSt16remove_referenceIT_E4typeE'/>
<fun r='_ZNK4llvm8Expected9toPointerEPKNSt16remove_referenceIT_E4typeE'/>
<fun r='_ZN4llvm8Expected9toPointerEPSt17reference_wrapperINSt16remove_referenceIT_E4typeEE'/>
<fun r='_ZNK4llvm8Expected9toPointerEPKSt17reference_wrapperINSt16remove_referenceIT_E4typeEE'/>
<fun r='_ZN4llvm8Expected10getStorageEv'/>
<fun r='_ZNK4llvm8Expected10getStorageEv'/>
<fun r='_ZN4llvm8Expected15getErrorStorageEv'/>
<fun r='_ZNK4llvm8Expected15getErrorStorageEv'/>
<fun r='_ZN4llvm8Expected12setUncheckedEv'/>
<fun r='_ZN4llvm8Expected15assertIsCheckedEv'/>
<mbr r='llvm::Expected::HasError' t='bool'/>
